server/plugin/authorization.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package plugin
+
+import (
+	"strings"
+)
+
+// isAuthorized returns true if the user is authorized to use the workflow plugin's admin-level APIs/commands.
+func (p *Plugin) IsAuthorizedAdmin(mattermostID string) (bool, error) {
+	user, err := p.API.GetUser(mattermostID)
+	if err != nil {
+		return false, err
+	}
+	if strings.Contains(user.Roles, "system_admin") { // || p.userAuthorized(mattermostID) {
+		return true, nil
+	}
+	return false, nil
+}
+
+// func (p *Plugin) userAuthorized(userName string) bool {
```

Comment:
Is the commented out code in this file intentionally left here for future use, or leftover debug code?